src/jobs/java_isopod_build_push_image.yml

```diff
@@ -0,0 +1,56 @@
+description: >
+  Build and push docker image of java app using isopod (leveraging prebuilt sources from caches and workspace).
+  Supports app-module from monorepo as well as single app from single-app-repo.
+
+parameters:
+  appname:
+    description: "Name of maven module for the app, or blank for single-app-repo"
+    type: string
+    default: ""
+  docker_hub_username:
+    type: string
+    default: $DOCKER_HUB_USERNAME
+  docker_hub_password:
+    type: string
+    default: $DOCKER_HUB_PASSWORD
+  private_hub_username:
+    type: string
+    default: $DOCKER_JFROG_USERNAME
+  private_hub_password:
+    type: string
+    default: $DOCKER_JFROG_PASSWORD
+  private_hub_url:
+    type: string
+    default: $DOCKER_JFROG_REGISTRY_URL
+  isopod_version:
+    description: "Isopod version to use for the build"
+    type: string
+    default: $ISOPOD_VERSION
+  docker_version:
+    description: "Docker version to use for the build"
+    type: string
+    default: "19.03.13"
+
+executor:
+  name: isopod
+  tag: << parameters.isopod_version >>
+  username: << parameters.private_hub_username >>
+  password: << parameters.private_hub_password >>
+
+steps:
+  - checkout
+  - maven_auth
+  - setup_remote_docker:
+      docker_layer_caching: false
```

Comment:
Why false?

Reply:
this is inline+customize of existing job [build_push_image](https://github.com/ricardo-ch/ricardo-orbs/blob/feat-java-orb/src/jobs/build_push_image.yml#L76-L78) - and there this is the default

since there was no other configuration in incidents-api config i went with this
should we need to make this configurable, i will extract this into a param with default=false (like in build_push_image job)